src/main/java/edu/kis/powp/jobs2d/drivers/transformators/ScalingDecorator.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.powp.jobs2d.drivers.transformators;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ScalingDecorator extends TransformationsDriverDecorator {
```

Comment:
I think that Movement (ShiftingX and Y by a constant) could be also useful. Please add such transformation.